packages/fa-icon-chooser/src/components/fa-icon-chooser/fa-icon-chooser.tsx

```diff
@@ -611,7 +611,27 @@ export class FaIconChooser {
       override = !!svgEmbedOverrideCallback();
     }
 
-    return override || this.embedSvgPrefixes.has(prefix);
+    return override || [...this.embedSvgPrefixes].length > 0;
+  }
+
+  emitIconChooserResult(iconDefinition: IconDefinition) {
+    const { prefix, iconName } = iconDefinition;
+    const iconLookup = { prefix, iconName };
+
+    const embedProSvg = get(this.kitMetadata, 'permits.embedProSvg', []);
+
+    // default to the restrictive case
+    let result = iconLookup;
+
+    const embedAllowed = this.shouldEmitSvgData() && embedProSvg.length > 0;
```

Comment:
If you were to revert the change to the `shouldEmitSvgData()` function above (so it works the way it did before), then I think this simply becomes:

```suggestion
    const embedAllowed = this.shouldEmitSvgData(prefix);
```

Reply:
I think you may not have been seeing the emitted SVG data for Pro Lite cases because the `&&` here would have undermined the override. Because `embedProSvg.length > 0` _basically means_ "not Lite", your code is saying `embedAllowed` will always be `false` when it's Lite, no matter what `shouldEmitData()` says, so also no matter what the override says.